root/static/scripts/release-editor/fields.js

```diff
@@ -1232,6 +1274,18 @@ class Release extends mbEntity.Release {
            !mediums[0].tracksUnknownToUser();
   }
 
+  shouldHaveLowercaseETI() {
```

Comment:
I might rename this to something like `shouldUseEnglishCapitalization` so I can share it with a future change to try to warn about other errors like "A", "Of", "The", etc. in the middle of titles.